docs/faq.md

```diff
@@ -24,39 +24,10 @@ View Models are a Redux-specific extension to these mechanisms. View models are
 **A**: Implement a [custom Read Model](read-side.md#custom-read-models). Custom Read Models allow you to use custom logic to communicate with a Read Model store.
 
 **Q**: How to send an aggregate command?<br/>
-**A**: To send a command from a client browser, use the [standard HTTP API](frontend.md#http-api): <br/>
-
-```
-$ curl -X POST "http://localhost:3000/api/commands"
---header "Content-Type: application/json" \
---data '
-{
-  "aggregateName":"Todo",
-  "type":"createItem",
-  "aggregateId":"root-id",
-  "payload": {
-    "id":`date +%s`,
-    "text":"Learn reSolve API"
-  }
-}
-```
+**A**: To send a command from a client browser, use the reSolve HTTP API or one of the available client libraries. Refer to the [Frontend](frontend.md) article for more information.
 
 On the server side, you can [send a command](api/saga.md#executecommand) from an API Handler or Saga:
 
-<!-- prettier-ignore-start -->
-
-[mdis]:# (../tests/saga-sample/saga.js#execute)
-```js
-await sideEffects.executeCommand({
-  aggregateName: 'User',
-  aggregateId: event.aggregateId,
-  type: 'requestConfirmUser',
-  payload: event.payload,
-})
-```
-
-<!-- prettier-ignore-end -->
-
 **Q**: How to perform validation to ensure input values are unique?<br/>
 **A**: In a distributed application, it is not possible to perform reliable checks. You should detect value duplicates in a Read Model or Saga projection code and mark duplicated values as incorrect.
 
```

Comment:
Please change line 63 below:

Line 63:
**Q**: How to implement a frontend?<br/>

**Q**: How do I implement a frontend?<br/>

Reply:
Done